func-complete-check/check-marker/index.ts

```diff
@@ -1,8 +1,28 @@
 import { AzureFunction, Context } from '@azure/functions'
 import { MarkCheckMessageV1 } from '../typings/message-schemas'
+import { performance } from 'perf_hooks'
+const functionName = 'check-marker'
+import * as V1 from './check-marker.v1'
 
 const serviceBusQueueTrigger: AzureFunction = async function (context: Context, markCheckMessage: MarkCheckMessageV1): Promise<void> {
-  context.log.warn(`check marking not yet implemented`)
+  const start = performance.now()
+  const version = markCheckMessage.version
+  context.log.info(`${functionName}: version:${version} message received for checkCode ${markCheckMessage.checkCode}`)
+  try {
+    if (version !== '1') {

```

Comment:
I am slightly concerned that the number could be passed as a number rather than a string.  Perhaps we could mitigate it with:

```suggestion
    if (Number(version) !== 1) {
```

Reply:
this is the same problem i encountered with the other TS functions - if i type it as a number, the binding/message still brings it in as a string directly off the message.  Happy to look at workarounds as it doesn't feel right.